apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.js

```diff
@@ -3,11 +3,212 @@
+    if (indentation >= 0 && indentation <= maxIndent) {
+      block.style.marginLeft = indentation * TABWIDTH + 'px';
+    }
+  }
+
+  function initializeBlockEvents(block) {
+    function removeSelectedAttribute() {
+      block.classList.remove('pl-order-blocks-selected');
+    }
+
+    function handleKey(ev, block, handle, focus = true) {
+      // When we manipulate the location of the block, the focus is automatically removed by the browser,
+      // so we immediately refocus it. In some browsers, the blur event will still fire in this case even
+      // though we don't want it to, so we temporarily remove and then reattach the blur event listener.
+      block.removeEventListener('blur', removeSelectedAttribute);
+      handle();
+      ev.preventDefault();
+      block.addEventListener('blur', removeSelectedAttribute);
+      correctPairing(block);
+      if (focus) {
+        block.focus();
+      }
+      setAnswer();
+    }
+
+    function handleKeyPress(ev) {
+      const optionsBlocks = Array.from($(optionsElementId)[0].querySelectorAll('.pl-order-block'));
+      const dropzoneBlocks = Array.from(
+        $(dropzoneElementId)[0].querySelectorAll('.pl-order-block'),
+      );
+      if (!block.classList.contains('pl-order-blocks-selected')) {
+        const moveBetweenOptionsOrDropzone = (options) => {
+          if (options && inDropzone(block) && optionsBlocks.length) {
+            optionsBlocks[0].focus();
+          } else if (!options && !inDropzone(block) && dropzoneBlocks.length) {
+            dropzoneBlocks[0].focus();
+          }
+        };
+        const moveWithinOptionsOrDropzone = (forward) => {
+          if (forward) {
+            if (inDropzone(block)) {
+              const blockIndex = dropzoneBlocks.indexOf(block);
+              if (blockIndex < dropzoneBlocks.length - 1) {
+                dropzoneBlocks[blockIndex + 1].focus();
+              }
+            } else {
+              const blockIndex = optionsBlocks.indexOf(block);
+              if (blockIndex < optionsBlocks.length - 1) {
+                optionsBlocks[blockIndex + 1].focus();
+              }
+            }
+          } else {
+            if (inDropzone(block)) {
+              const blockIndex = dropzoneBlocks.indexOf(block);
+              if (blockIndex > 0) {
+                dropzoneBlocks[blockIndex - 1].focus();
+              }
+            } else {
+              const blockIndex = optionsBlocks.indexOf(block);
+              if (blockIndex > 0) {
+                optionsBlocks[blockIndex - 1].focus();
+              }
+            }
+          }
+        };
+        switch (ev.key) {
+          case 'Enter':
+            handleKey(ev, block, () => block.classList.add('pl-order-blocks-selected'));
+            break;
+          case 'ArrowUp':
+            handleKey(
+              ev,
+              block,
+              () => {
+                moveWithinOptionsOrDropzone(false);
+              },
```

Comment:
Nit:
```suggestion
              () => moveWithinOptionsOrDropzone(false),
```
same for other similar cases below.